.github/workflows/diffcalc.yml

```diff
@@ -182,6 +201,10 @@ jobs:
               fi
           done
 
+      - name: Add master environment
+        run: |
+          sed -i "s;^OSU_A=.*$;OSU_A=${{ needs.master-environment.outputs.HEAD }};" "${{ needs.directory.outputs.GENERATOR_ENV }}"
```

Comment:
If you want to check that this is correct, refer to the step below this one (and note `B` -> `A`).